src/pages/trade-types/accumulators/_how-accumulators-works.tsx

```diff
@@ -1,27 +1,27 @@
 import React from 'react'
 import { graphql, useStaticQuery } from 'gatsby'
 import styled from 'styled-components'
-import { SmallContainer, Grid, HowItWorksItem } from '../components/_style'
+import { SmallContainer } from '../components/_style'
 import SideTab from '../components/_tabs'
 import CommonHeaderSection from 'components/elements/common-header-section'
-import { SectionContainer, Flex, Container } from 'components/containers'
-import { Header, Text, QueryImage } from 'components/elements'
+import { SectionContainer, Container } from 'components/containers'
+import { Header, QueryImage } from 'components/elements'
 import { localize, Localize } from 'components/localization'
-import device from 'themes/device'
-import Pattern from 'images/common/trade-types/pattern-section.png'
 
 const query = graphql`
     query {
         accumulator_market: file(relativePath: { eq: "trade-types/accumulators-market.png" }) {
             ...fadeIn
         }
-        option_trade_type: file(relativePath: { eq: "trade-types/options-trade-type.png" }) {
+        accumulators_trade_type: file(
+            relativePath: { eq: "trade-types/accumulators-trade-type.png" }
+        ) {
```

Comment:
Any chance to refactor images fetching by new approach? 🙏 https://deriv-group.slack.com/archives/C02CTVA0LBS/p1677037272787159